### ISSUE TITLE
Adding license file to some of the BERT models

### ISSUE THREAD

**Issue by sasaadi** [huggingface/transformers#15862]
Hi,

I am interested in using “[distilbert-base-german-cased](https://huggingface.co/distilbert-base-german-cased)”, “[bert-base-uncased](https://huggingface.co/bert-base-uncased)”, and “[distilbert-base-uncased](https://huggingface.co/distilbert-base-uncased)” models for industrial applications. To be able to download these models from Hugging Face and use them in applications, the stated “apache-2.0” license in the readme files (https://huggingface.co/distilbert-base-german-cased/blob/main/README.md) is unfortunately not sufficient and an explicit license text file (i.e., [text for the Apache-2.0 license](https://opensource.org/licenses/Apache-2.0)) with copyright information is required to avoid ambiguity.
 
I would like to know if you could add a license file with copyright information to these models on Hugging Face.
 
Thank you.

**Comment by sasaadi** [comment]
Hi @VictorSanh, I would like to know if a license file can be added to the above mentioned models. Since you have proposed the Distilbert models, I was just wondering if you could maybe provide a license file for the Ditsilbert models? 

Thank you.

**Comment by LysandreJik** [comment]
Interesting question regarding lincense of models on the hub cc @osanseviero @julien-c

**Comment by julien-c** [comment]
Yes we can add the full-text version of the licenses to those 3 models, I see no objection to doing it.

Curious though, could you expand on 
> unfortunately not sufficient and an explicit license text file [...] with copyright information is required to avoid ambiguity.

Is this a legal requirement from your company? Would you have more details to share? Thanks! cc @annatrdj

**Comment by sasaadi** [comment]
Thanks a lot @julien-c for the reply. As stated in the [Apache 2.0 license §4](https://opensource.org/licenses/Apache-2.0), recipients must get a copy of the license and therefore, we require a copy of the license for the models. 
I appreciate your effort and would be grateful if you could add the license files to these models.

Thank you.

**Comment by VictorSanh** [comment]
> Yes we can add the full-text version of the licenses to those 3 models, I see no objection to doing it.

sgtm!

**Comment by sasaadi** [comment]
Hi @julien-c, I would like to know if there is any update on the lisence files? 

Thank you.

**Comment by sasaadi** [comment]
Hello @julien-c, is there any updates on the this topic?
Thanks.

**Comment by julien-c** [comment]
this is still in progress but hoping to have an update on this in the coming month

**Comment by julien-c** [comment]
OK we now have a Hub Pull request feature which is a good use case for this.

Please check the PR to add the full-text of the license to [`distilbert-base-uncased`](https://huggingface.co/distilbert-base-uncased/discussions/1)

you can suggest PRs to add the full-text to other models as well, but FYI we'll also work on a UX feature to easily get to the full text of the license from the license tag of any repo + a capability to download it programmatically. It should comply with the text and the spirit of the licenses.

In the meantime feel free to suggest other PRs if necessary

**Comment by sasaadi** [comment]
Thanks for the update. I appreciate your effort. This will be very useful. Then I make a PR for the other model that I mentioned in the post earlier.
